client/my-sites/controller.js

```diff
@@ -254,12 +273,11 @@ function onSelectedSiteAvailable( context ) {
 	}
 
 	/**
-	 * The paths allowed for domain-only sites and DIFM in-progress sites are the same.
 	 * Ignore this check if we are inside a support session.
 	 */
 	if (
 		isDIFMLiteInProgress( state, selectedSite.ID ) &&
-		! isPathAllowedForDomainOnlySite(
+		! isPathAllowedForDIFMInProgressSite(
 			context.pathname,
 			selectedSite.slug,
 			primaryDomain,
```

Comment:
This if condition is now a bit confusing, maybe we can move isSupportSession inside the first condition and rename it to reflect that (isDIFMLiteBlockedOutView or some other better name if you can think of one)

Reply:
Thanks, that's good feedback! I intended to follow the style already used in the same function that handles the domain-only screen's rendering. I have updated the comment and changed the order to make it a bit easier to read.